inc/TRestGeant4Event.h

```diff
@@ -492,11 +492,11 @@ class TRestGeant4Event : public TRestEvent {
     TPad* DrawEvent(TString option = "") { return DrawEvent(option, true); }
     TPad* DrawEvent(TString option, Bool_t autoBoundaries);
 
-    // Construtor
+    // Constructor
     TRestGeant4Event();
     // Destructor
     virtual ~TRestGeant4Event();
 
-    ClassDef(TRestGeant4Event, 5);  // REST event superclass
+    ClassDef(TRestGeant4Event, 6);  // REST event superclass
```

Comment:
There is no reason to increase class version here! The data members did not change.

Reply:
True, my bad, I was thinking on the future changes in this PR already